server/queues/tasks/DeliverWebhookTask.ts

```diff
@@ -180,6 +183,11 @@ export default class DeliverWebhookTask extends BaseTask<Props> {
       case "shares.revoke":
         await this.handleShareEvent(subscription, event);
         return;
+      case "subscriptions.create":
```

Comment:
I appreciate the thoroughness, I don't think we need these as webhook events though – to me a document subscription is basically a personal setting akin to starring a document. There isn't value in broadcasting this, in that vein we might not even need to record events for it at all.